src/perf/index.md

```diff
@@ -69,14 +74,15 @@ the quicker it is to download.
 
 {% comment %}
 
+TODO: Reintroduce this article and add this link back.
```

Comment:
Do you want to commit TODO items?

Reply:
It's a comment and I've added links to the issues now, so I think it should be fine. It will make it easier to notice that it's the section that needs to be added back.